app/views/feedbacks/_form.html.haml

```diff
@@ -1,37 +1,5 @@
 %script{src: "//www.google-analytics.com/cx/api.js?experiment=4uAdObjoRp2Cz3Ve100AvA"}
-%noscript
-  %p If you need to ask a question about: 
-
-  %ul
-    %li whether your visit has been booked 
-    %li cancelling or changing visit details eg names of visitors, dates or times
-    %li what you can bring when you visit, eg clothes or money 
-    %li prisoner's contact lists or visiting allowance
-
-  %p
-    You must 
-    %a(href="http://www.justice.gov.uk/contacts/prison-finder") contact the prison
-    for help. 
-
-  %p You can use this form to tell us about a problem you’re having with the visit request website, or to ask us a question about how to use it. 
-
-.variant.a.hidden
-  %p If you need to ask a question about: 
-
-  %ul
-    %li whether your visit has been booked 
-    %li cancelling or changing visit details eg names of visitors, dates or times
-    %li what you can bring when you visit, eg clothes or money 
-    %li prisoner's contact lists or visiting allowance
-
-  %p
-    You must 
-    %a(href="http://www.justice.gov.uk/contacts/prison-finder") contact the prison
-    for help. 
-
-  %p You can use this form to tell us about a problem you’re having with the visit request website, or to ask us a question about how to use it. 
-
-.variant.b.hidden
+.experiment-1-outcome-b
```

Comment:
I'll remove that. I thought it might be helpful to have that in the markup for later reference, but maybe that wasn't a good idea.